test/tools/README.md

```diff
@@ -28,7 +28,17 @@ export TEST_PROVISION_FILE=${PROPERTIES_FILE_PATH}
 export TEST_PROVISION="yes"
 ./caa-provisioner-cli -actions=${ACTION}
 ```
-`ACTION` supports `provision`, `deprovision` and `uploadimage`.
+`ACTION` supports `provision`, `deprovision`, `deploy`, `delete` and `uploadimage`.
+
+### Brief Action Explanations
+
+`provision` : Uses the provisioner to create a cluster
+
+`deprovision` : Deletes the cluster that we previously created
+
+`deploy` : Deploy the cloud-api-adaptor using the operator to an existing cluster, must set `KUBECONFIG`
+
+`delete` : Removes the cloud-api-adaptor resources from the cluster, must set `KUBECONFIG`
```

Comment:
I wonder if something like undeploy/uninstall might be better than delete for this action as you might expect the delete of the cluster to be done under that action?

Reply:
We use `Uninstall CoCo and cloud-api-adaptor` in the info message, so maybe `uninstall` would be best?

Reply:
I based it off of the `make deploy` and `make delete` already in use. I agree it isn't the best naming though

Reply:
Changed to `install` and `uninstall`